index.bs

```diff
@@ -4747,6 +4855,9 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 1. If [=user verification=] is required for this assertion, verify that the [=User Verified=] bit of the <code>[=flags=]</code> in
     |authData| is set.
 
+1. If the credential [=backup state=] is used as part of Relying Party business logic or policy, compare the previously stored 
+    value with the [=backup state=] bit of the <code>[=flags=]</code> in |authData|, perform evaluation, and then store the new value.
```

Comment:
```suggestion
1. If the credential [=backup state=] is used as part of [=[RP]=] business logic or policy, compare the previously stored 
    value with the [=backup state=] bit of the <code>[=flags=]</code> in |authData|, perform evaluation, and then store the new value.
```

Reply:
Also, up on line 3728 we recommend the RP storing both flags/bits, but here in the RP ops sections we are recommending storing only the "backup state" and not "backup eligibility".